backend/auth.py

```
@@ -0,0 +1,55 @@
+            "message": "Login successful",
+            "token": JWT_SECRET  # SECURITY ISSUE - Exposing secret as token
+        }
+    
+    raise HTTPException(status_code=401, detail="Invalid credentials")
+
+def validate_token(token: str):
+    """Validate JWT token - SECURITY VULNERABILITY"""
+    # SECURITY ISSUE - No actual JWT validation, just string comparison
+    if token == JWT_SECRET:
+        return True
+    return False
```

Ruff (0.16.1)

[warning] Return the condition `token == JWT_SECRET` directly

Replace with `return token == JWT_SECRET`

(SIM103)